data/composition/52.json

```diff
@@ -150,38 +172,35 @@
         "province": "Brussel-Halle-Vilvoorde",
         "language": "Nederlands",
         "wiki": "https://nl.wikipedia.org/wiki/Ben_Weyts",
-        "replaces": [{
-            "name": "Van Rompuy Herman",
-            "dates": [
-                {
-                    "from": "2008-12-31"
-                }
-            ]
-        }]
+        "replaces": [
+            {
+                "name": "Van Rompuy Herman",
+                "dates": [
+                    {
+                        "from": "2008-12-31"
+                    }
+                ]
+            }
+        ],
+        "date_of_birth": "1970-11-12"
     },
     {
         "first_name": "Michel",
         "last_name": "Doomst",
         "party": "CD&V",
         "province": "Brussel-Halle-Vilvoorde",
         "language": "Nederlands",
-        "wiki": "https://nl.wikipedia.org/wiki/Michel_Doomst"
+        "wiki": "https://nl.wikipedia.org/wiki/Michel_Doomst",
+        "date_of_birth": "1953-11-13"
     },
     {
         "first_name": "Sonja",
         "last_name": "Becq",
         "party": "CD&V",
         "province": "Brussel-Halle-Vilvoorde",
         "language": "Nederlands",
-        "wiki": "https://nl.wikipedia.org/wiki/Sonja_Becq"
-    },
-    {
-        "first_name": "Carl",
```

Comment:
To clarify: this entry was moved to the bottom of the file because this was one I did manually because of edge case